natgeo/mortar
# [AEM-6520] Using a white border instead of margin to add space before first POD promo

https://jira.natgeo.com/browse/AEM-6520

Using a white border instead of margin to add space before first POD promo on the page to work around fixed positioning woes in Firefox.

To test:
npm test
gulp serve
verify component renders

Get this updated file into AEM, build platform-view and verify change mitigates the display issues in Firefox.
